doc/check.texi

```diff
@@ -1217,6 +1217,17 @@ would pass.
 @item ck_assert_ptr_eq
 @itemx ck_assert_ptr_ne
 
+Compares a pointers against null and displays predefined message with
+condition and value of the input parameter on failure.
+@code{@item ck_assert_ptr_null} checks that pointer is equal to NULL and
```

Comment:
I do not think you need the @item inside the @code{}. This may be what the build is complaining about when checking the documentation.